src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1125,6 +1130,11 @@ namespace winrt::TerminalApp::implementation
                     tab->ExitZoom();
                 }
 
+                if (tab->GetLeafPaneCount() == 2)

```

Comment:
This _feels_ like the wrong time to do this check. Is there any place where we would know that there's only one pane left? 

There's an (admittedly very edge) case where you open 3 panes, then "move focus to the parent pane", then close pane, and now there's 1 pane left _and the menu item is visible_.